backend/amd64/arch.ml

```diff
@@ -334,3 +334,23 @@ let equal_specific_operation left right =
     | Ifloat_iround | Ifloat_round _ | Ifloat_min | Ifloat_max | Ipause
     | Icrc32q | Iprefetch _), _ ->
     false
+
```

Comment:
Isqrtf and Icrc32q are pure.

Reply:
That's what I thought, until I read https://github.com/ocaml-flambda/flambda-backend/blob/main/backend/amd64/proc.ml#L424.